### ISSUE TITLE
team.md needs updating - but how?

### ISSUE THREAD

**Issue by peabee** [puppylinux-woof-CE/puppylinux-woof-CE.github.io#30]
Section 2 of
https://github.com/puppylinux-woof-CE/puppylinux-woof-CE.github.io/blob/master/team.md
needs updating....
no @wdlkmpx !!!!!!!!!!
no @gyrog
no @woodenshoe-wi
all of whom contribute and have commit rights

@dimkr hasn't been seen for some long time
@01micko is "otherwise engaged" in real life so not seen very often

https://github.com/puppylinux-woof-CE/woof-CE/graphs/contributors

**Comment by mavrothal** [comment]
Isue a pull request.
While at it, update also the front page (index.md) with the 32 and 64bit BB downloads.
 
They where never announced anywhere as "official" though so you might want to coordinate Phil ( @mrfricks  ) and write a proper announcement for both 
Though the latest UpupBB might need a rebuild with comparable name and versioning scheme

**Comment by mavrothal** [comment]
Come to think of it, you have commit writes in that page.
Change Woof-CE stewards (I guess that is what we are discussing here) as you see fit !!!
I would actually be happy to be removed (which I do not do my self as the other 3 are not very present the lately) as soon as  some more active ones are added.

**Comment by jamesbond3142** [comment]
You definitely need to add those 3 names you suggested into the stewards' list. (Anyone who have commit rights to Woof-CE a steward by definition - except me :rofl: ). 

EDIT: If you have commit right peabee that you should put your name too, you're definitely making a lot of contributions too these days. Does 666philb (mrfricks) has commit right too?

As for the dimkr and others, you can create a new subgroup called "past stewards" or "retired stewards" or "inactive stewards" or something to that effect, to acknowledge their past contribution __and__ the fact that they're not actively present anymore. They can be moved back to the active stewards list anytime when they return back to activity.

For 01micko, he's busy right now but I still see him committing and approving stuff now and then, so I suppose although less often seen these days, he's still considered as an active steward.

What do all you think?

**Comment by mavrothal** [comment]
Hmmm, from what I can [see](https://github.com/orgs/puppylinux-woof-CE/people) neither Phil nor Peebee have commit writes in woof-CE. Interested???
One thing that new stewards may consider is to provide a real name (says me.... 👅  ) because if igu and ziggy "retire" only Mick will be identifiable!

**Comment by peabee** [comment]
I would welcome commit rights.... but not sure about the "real name"... inclusion of Forum name should be sufficient to make contact....
I can't see the list of people with commit rights.... but @jamesbond3142 is right they should all be mentioned - with any "sleepers" moved to the "retired" list

As for Bionic Pups - UPupBB and BionicPup64 are very different builds with only their compat distro base and Woof-CE in common really - so I think they should remain different so people don't think they are 32/64 bit versions of the same animal. I also like my versioning which has some meaning.
Announcing them as "Official" means sending something to Distrowatch by somebody who DW respect as being a Puppy spokesperson - only BarryK or @01micko  are in that category that I know of.... can we entice Mick to do that??

**Comment by jamesbond3142** [comment]
@peabee 
Mick is probably very busy until end of this week. I'm not sure he's checking his emails but if you have something very important I can pass a message to him. Don't expect immediate answer though.

**Comment by mrfricks** [comment]
@peabee

i posted xenialpup myself to distrowatch when it was released. If i remember correctly they wanted a direct link to the ISO, a good resolution screenshot (ie 1080 or 720) and a direct link to the release announcement.  

concerning your versioning of upup. whilst you like it, and the versioning might make sense for ubuntu based pups, it's not applicable to dpups & slackpups, it also makes a nonsense of all puppy versions to date. 
Another thing to take into account is that in the past distrowatch had issues with how puppy releases were versioned. I think this was probably in the lucid/precise v slacko days when a release with a lower number was released after one with a higher version number.

i personally think that it's not a problem that the 32bit and 64bit bionics are done by different people and are different as they are built for different systems & specs. It's a good thing actually as the different architecture get more attention. The only issues i see if both are to be released, is the names not matching and that the puppy version should be 8.0

the name issue is easily solved upupbionic AKA bionicpup

**Comment by peabee** [comment]
My ScPups (Slackware Current) have the same versioning - if and when Slackware-15 is released it would be my plan to do S15Pup based on the same build recipe and with the same versioning....but only if it looks unlikely that Slacko-15 is going to appear....

In retrospect I think I probably shouldn't have done UPupBB (& UPupCC) as I don't really like Ubuntu's way of organising its components (and will never do a 64-bit version). Whether I'll feel inclined for UPupDD to appear in the next month or so - who knows?! Also, the issued version of UPupBB (18.05+28 as we speak) is still based on the Master branch of Woof-CE and needs some work to change it to the current Testing branch of Woof-CE (as used by BionicPup64) and if I did this I'd probably update the kernel and give it a new 19.nn version as well.

Back to the issue of updating team.md - I can now see the members page courtesy of @mavrothal  - can we agree that @dimkr and @pemasu are "retired" "Stewards"? I am unsure of the status of @JakeSFR?

**Comment by woodenshoe-wi** [comment]
Please put me on the "inactive stewards" list.  I am worried that my hard drive may be failing so I'm not using my computer until I am ready to try to copy my files off of it, and I get rather busy in spring, summer and fall (northern hemisphere) with yard work and gardening.

**Comment by mavrothal** [comment]
I think that we should not confuse stewards and woof members. Stewards was coined because a) the the woof members list is not public and b) some woof members were either focusing elsewhere or were "forced"  to become members.  
We can either ask for "volunteers" to show as stewards or make woof member list public, point to it in the puppy home page  and forget about the stewards segment. Just leave Mick there alone 😈

**Comment by mavrothal** [comment]
Regarding the Ubuntu based Puppy8 I think that the difference between 32 and 64 bit versions is no problem as the 32bit can be promoted "for older hardware". Regarding the Update scheme, nothing says that bionicpup/UpupBB 8.0 can not follow the "+" update scheme instead of the "8.0.X" scheme while the pup64 the other. All it needs is to rename  the 18.05+28 iso and distro specs to $WHATEVER_YOU_DECIDE 8.0. 
Alternatively  you can only add them the official puppy site and take your case to distrowatch individually, if at all.

**Comment by peabee** [comment]
> We can ..... make woof member list public, point to it in the puppy home page and forget about the stewards segment. Just leave Mick there alone

This sounds like a good idea to me....

**Comment by peabee** [comment]
> Alternatively you can only add them the official puppy site and take your case to distrowatch individually, if at all.

This also sounds like the best way to proceed to me....

**Comment by peabee** [comment]
I should perhaps clarify my versioning.....even though it is OT ;-)
The version +n ONLY applies to the .iso - the component .sfs and DISTRO_SPECS within the .iso remain constant at the values in the first base .iso. Therefore new "versions" do NOT trigger a personal save update. The +n in the .iso name is there to allow updates to be distributed as xdeltas. I think it therefore follows that I can't satisfy the DistroWatch requirement for a direct link to the ISO..... (my .isos although "available" through ibiblio are actually stored on SourceForge)

**Comment by mavrothal** [comment]
@peabee 
You can obviously do as you see fit regarding UpupBB announcement. 
I just think that without a conforming version (7, 8, 9...) and being "announced" 10 months latter decreases the probability of acceptance in DW. Let's hope that will not be the case.
If indeed DW rejects it, announcing a 64bit-only v8.0 release for a distro labeled "for older hardware", might also decrease overall impact of puppylinux,  but hopefully not. 
Oh well, is your work after all.

**Comment by jamesbond3142** [comment]
> I think that we should not confuse stewards and woof members. Stewards was coined because a) the the woof members list is not public and b) some woof members were either focusing elsewhere or were "forced" to become members.

I would say that they're both the same. Woof-CE is what makes a puppy, so anyone who has the potential to make changes to Woof-CE (i.e., commiters to Woof-CE) is a steward - someone who is entrusted with the safekeeping and safeguarding of future Puppies. 

But I agree the point that not all the __current commiters__ are stewards. Me for example. I was granted commit access when we worked with this website and for woof-next (I think). The reason was to make things go faster at the time we were building the website. Beyond that I have never used my commit rights, because I'm not active in Puppy development (apart from giving advice/suggestion every now and then, which I'm happy to do). Now, woof-next is today a dead project and the website is established, so I would say that my commit access can be revoked. Although listed as a commiter I was and never has been a steward (unlike Mick or mav are). So if you're going to show the list of committers, it is still necessary to make the distinction unless you can be sure that everyone who is a committer is also a Puppy developer (either past or present).

What I have in mind when we coined the word "steward" was this: there will be many contributors to Woof-CE, which operates by the usual github pull request. Over time, we will know who are the prolific contributors, and contributors with enough responsibility not to whack the code when they make changes. These people should then be promoted to have commit rights; and when they do, they become the gatekeeper of reviewing contributions from others before merging them with the mainline (=and thus, become a steward). 

Of course, it is always a good practice for even a steward to issue a pull request first when making changes (because at this point he/she is acting as a developer and not as a steward), and let other stewards see the changes that he/she is proposing. This way nobody is stepping on everyone else's toes. Of course, you can make exceptions - for example right now wdlkmpx is probably the one who knows left and right about Puppy, so he can be excepted to make direct commits, etc.

> We can either ask for "volunteers" to show as stewards or make woof member list public, point to it in the puppy home page and forget about the stewards segment. Just leave Mick there alone.

We'll we need to have at least two names, to cover for the occasion when the master steward isn't available. I think we still need mavrothal listed there too :see_no_evil: 

> Back to the issue of updating team.md - I can now see the members page courtesy of @mavrothal - can we agree that @dimkr and @pemasu are "retired" "Stewards"? 

I haven't seen them for a very long time both here and on the forum. I think they have other interest now and they can be safely re-assigned into the "retired" category.

> I am unsure of the status of @JakeSFR?

JakeSFR is now a member of Fatdog team. He still does and makes general contributions to Puppies in general, but I'm not sure of his time availability or commitment to Woof-CE. You can still find him in the forum, so it's best to ask him. I will also ask him personally and hopefully he can respond himself here.

**Comment by JakeSFR** [comment]
> > I am unsure of the status of @JakeSFR?
> 
> JakeSFR is now a member of Fatdog team. He still does and makes general contributions to Puppies in general, but I'm not sure of his time availability or commitment to Woof-CE. You can still find him in the forum, so it's best to ask him. I will also ask him personally and hopefully he can respond himself here.

Hey guys!

Yeah, I haven't been involved in development of Woof-CE for a long time, so I think I should be moved to the "retired" group. :)

**Comment by mavrothal** [comment]
> We'll we need to have at least two names, to cover for the occasion when the master steward isn't available. I think we still need mavrothal listed there too 🙈

Let me think.... let me think .... let me think....
Got it!
**NO**

**Comment by woodenshoe-wi** [comment]
jamesbond3142 wrote:
> What I have in mind when we coined the word "steward" was this: there will be many contributors to Woof-CE, which operates by the usual github pull request. Over time, we will know who are the prolific contributors, and contributors with enough responsibility not to whack the code when they make changes. These people should then be promoted to have commit rights; and when they do, they become the gatekeeper of reviewing contributions from others before merging them with the mainline (=and thus, become a steward).

I didn't know being a member with commit rights also involved becoming a gatekeeper.  I have a good understanding of the inner working of the woof scripts, but don't want to be part of the decision making process.

I don't mean to insult anyone by leaving the organization, but I don't want to be confused with the actual leadership team.

**Comment by jamesbond3142** [comment]
> I didn't know being a member with commit rights also involved becoming a gatekeeper. I have a good understanding of the inner working of the woof scripts, but don't want to be part of the decision making process.
> 

When you've got commit rights then it means you're trusted. "Trusted" because you have the ability to directly influence what happens to Woof-CE (and thus, Puppy). In another word, you have been "entrusted" that you will do the best for Puppy.  A person who is "entrusted" for something which is not his/hers, is a steward, by definition (English word).

By being a steward, you also have ability to review other people's submissions (committer or non-commiters) ==> you have the ability to become a gatekeeper. Whether or not you choose to exercise this ability is really up to you. If you don't like it, then don't do it. Hopefully someone else will pick the tab. Because obviously you can't have all the committers commit to the same tree left and right.

> I don't mean to insult anyone by leaving the organization, but I don't want to be confused with the actual leadership team.

There is no leadership team. You have been here long enough to know that. The leadership team is the people who contributes to Puppy. It has always been like that since day one. Nobody is in charge of making any decisions of what happens. It's always a consensus, stewards or no stewards (committers or no committers). Good idea eventually gets merged, bad idea gets rejected. That's it. That's why Woof-CE has the suffix "CE" to it. (CE=Community Edition).

All these things about stewardship is:
a) so that people know who are the active contributors
b) so that people know who to contact for Woof-CE issues
c) to a lesser extent, so that multiple committers don't step on each other's toes.

---

Anyway, that are just points for discussion. I am not an active Puppy developer. I speak without any authority. I just happened to have been here long enough to be able to give the background for all these terms "stewards" and what is idea behind it. At the end of the day, it's the people who are actively contributing who gets to decide the direction they want to take. People like you, not people like me. 

If you don't like the idea or the word "steward", then by all means change it. If you all agree to remove the word "stewards" and replace it with "committers" - then so be it. If you all agree otherwise, then so be it too. If you want to coin another term, go for it. If you want to a clearly defined responsibilities for each, go for it. If you want to operate on loose terms, that would do, too.

I just have one last word to say. Based on the words in the Puppy forum, I would say that Puppy users generally want to have transparency. They want to know who are the people behind the schemes, because transparency encourages responsibility and communication. Too often in the past Woof-CE has been wrongly blamed as "exclusive" and "impenetrable fortress". Or "who are these mysterious men-in-black developing Puppy nowadays" or something to that effect. This "team" page is an attempt to de-fuse such questions, but as time passes, it has become obsolete and in need of an update.

How you want to update it is really up to you. After all, you are all Puppy team :smile:

**Comment by peabee** [comment]
Here is my suggestion for changes. What do people think?
========================
List of current Puppy Linux Stewards

1. Puppy Linux Master Steward - Mick Amadio (01micko).
Role: Confirm which distribution is considered as an “Official Release”. Responsible for the puppylinux.com domain as well as the main Puppy Linux distribution site on ibiblio.org

2. Other Woof-CE Stewards - people who:
a. are currently active in the Puppy Linux Community
b. have been given Commit Rights to the Woof-CE GitHub repository

The best way to contact the various Stewards is to:
a. look at the Woof-CE GitHub Contributors page https://github.com/puppylinux-woof-CE/woof-CE/graphs/contributors and the latest commits https://github.com/puppylinux-woof-CE/woof-CE/commits
b. locate the Steward on the Forum (most people have the same or similar identities on GitHub and on the Forum) and then either send a Private Message (PM) or post an entry on a suitable Forum Thread http://murga-linux.com/puppy/viewtopic.php?t=101174

**Comment by mavrothal** [comment]
Let's keep it simple.
Stewards are gone (but the head honcho). 
In reality no-one checks/approve code or "set" directions and priorities. So stewards have to practical meaning.

Regarding the woof-ce contributors, we can either
a) just list  woof members in the puppylinux.com page
or 
b) list visible members and number of non visible members (ie "me, you, him, her and 5 more private members -login in github to see them"). 
For option (b) current woof members go to their  entry in member list, select "manage" from the dropdown menu on the right in their line  and turn their membership from private to public.
And since if we go for (b) no one is going to bother the assumption is that **you all want to be public** unless you state otherwise by March 30!

Personally I go for option (a) ie listing current members without links, profiles, etc. Just the handle and name if visible in github.
Retired, on sabbatical, etc has no point either. As members they can still do anything in woof-ce, even if they do not. If they have no intension  to ever do anything they can just leave the woof-ce "organization" (hopefully non will).

Regarding new woof-ce members the call is always open, but I have not seen major influx. (Even Phil besides the direct invitation above showed no interest .. 😞  ). Based on past contribution @rserwin1 and  @ninaholic could be invited (not sure if ever did in the past) but I can not see many more.
A reminder in the forum though could be useful.

BTW, offering "stewardship" to active/vocal forum members (assuming the stewards remain), I'm afraid will transfer the havoc of the forum in github.

**Comment by mrfricks** [comment]
hi mavrothal,
i missed that invite! i would definitely like to be an official member of woofce.

**Comment by jamesbond3142** [comment]
> In reality no-one checks/approve code or "set" directions and priorities. So stewards have no practical meaning.

I'm assuming that Mick did this in the past (or in the present when he's not too busy), and I think wdlkmp plays that role now, more or less.

If nobody checks anything and every committers commit directly with no supervision we'll have chaos and breakage very soon. Even in the LInux kernel you have one guy who's approving commits to the mainline (previously only Linus, but now Greg also has commit rights). Since we already have multiple committers to the mainline, this is something that must be agreed among yourselves. 

It's off-topic for this discussion, of course, but something that must be agreed sooner or later. May be open another issue, if you think it's needed. Or else we can let the train continue as is and only deal with the problem when it actually happens. May be better that way, less discussion and more code instead :tongue: 

> BTW, offering "stewardship" to active/vocal forum members (assuming the stewards remain), I'm afraid will transfer the havoc of the forum in github.

No, please. Stewardship (or commit rights - depending on how you see it) - is earned. It's not given or offered for anyone. Everyone is on the Puppy team, and they can contribute via pull requests (which will get reviewed before merging) or leaving comments. But not everyone can change bits and pieces of Woof-CE anytime and anyhow they like. Let them do that a couple of times. If their pull requests are good (careful, clean, easy-to-read, and not breaking anything), and they are frequent enough contributors , then the current committers (=current stewards) may want to extend the invitation to them. Otherwise, they can continue to contribute via pull request. 

We welcome contributions but we don't welcome chaos. We are open for contributions but not everyone can change cogs and gears in Woof-CE directly (they're welcome to clone Woof-CE and do it in their own private copy), not until they can be trusted that they would do the right thing.

EDIT: Commit rights isn't about how vocal someone is or how how frequent they make comments in the forum. It's about the ability to code, and to code well.

**Comment by wdlkmpx** [comment]
It's clear that the the project is driven by those who understand the
code and make improvements, cleanups, fixes, and implement new
features, etc.. this is not coordinated, but it could be. In a perfect
world.

There is something to understand here, there are very conflicting
views inside woofce, iguleder said to hell with woofce when a
defaults-chooser (now puppyapps) was being implemented .. he said he
was sick of ptheme and so on.

I know people have read my many rants over the years, my hatred of
rox, the 2barks, and basically all the work in creating a complete DE
and trivial apps instead of using what's already available, the distro
name itself, etc. I'm not fond of canines.. I'm more into felines.

However what i did is the opposite, i improved and fixed and even
simplified many woofce apps, i broke & fixed rox-reladted stuff, etc.
But all these changes actually make it easier to fork woofce.

So overall, i've been trying to move my agenda forward while making
meaningful contributions to woofce. I did this because i like the idea
of contributing to something greater, i don't release isos, i don't
enjoy the attention either, so my fork would be completely unknown,
that's why.

**Comment by wdlkmpx** [comment]
Back on topic. I'm not sure about what to think about who are the
stewards but i guess we all are busy with are our lives and code, but
i do read the code of each pull request, and analyze the situation,
every single line. Sometimes i don't test it, but if i don't see
something terribly wrong, i just merge it. It looks like if i don't,
people will to have to wait weeks before micko decides to close it or
merge it.

rerwin should be granted write accesss, ninaholic should receive at
least a honorary mention

peabee should declare all of his upup versions deprecated and focus
only on slacko and even release slacko 700 on behalf of micko.

Debian will introduce a small incompatibility:
https://wiki.debian.org/UsrMerge

It will only affect pet packages and alien packages, but woofce is
somewhat fixed or is easily fixed for that event.. thanks to all the
changes in packages-templates and 2createpackages

and etc ..

**Comment by wdlkmpx** [comment]
I guess those who have stopped contributing for more than 2 years can
be removed from the "team" list or something, except jamesbond and the
owner of the repo.

woof-next is a very significant improvement and simplification, having
both methods working with the current rootfs-skeleton and init ensures
puppy can still use any compat distro and at the same be "extremely"
compatible with a least one of them..

I guess i can bring woof-next to testing, i don't know if i'll be able
to success, but i'll try.

**Comment by mavrothal** [comment]
And after all this talk we are here!  15da044
As long as we all have commits rights anybody can change the page as long as no personal information is revealed other than what it is in the [woof people](https://github.com/orgs/puppylinux-woof-CE/people) list

Now regarding "Puppy 8" I'm afraid is between Phil and PB and there is not much that we can do.

**Comment by wdlkmpx** [comment]
When i said peebee should declare all his upup versions deprecated, i
didn't mean it in a bad way, i think he produces traditional pups that
are completely aligned with woofce and when i tell someone to test a
puppy ISO, it must be one of peebee's isos.

Proper o acceptable compatibility with ubuntu/debian is a complex
issue that will require more work and fixes... and this will make
peebee's life a nightmare....

**Comment by mavrothal** [comment]
@woodenshoe-wi I do not know if the current wording about woof members (not stewards any more) in the puppylinux page eases your concerns, but if it does I would be happy to reinvite you back.

 at all: closing, reopen as needed

**Comment by woodenshoe-wi** [comment]
Some years ago I worked on the woof-CE build scripts to get the raspup build working.  At the time I wasn't a member and I put the changes in my fork of the woof-CE repo on my GitHub page.

When wdlkmpx noticed them I helped get them merged into the newer woof-CE scripts because things had changed in between when I first got the raspup build working and wdlkmpx found my changes.

I also worked on a compiled helper program to speed up the findpkgs script.

These projects have kind of come to a conclusion, the raspup build is merged and I haven't heard of any problems with findpkgs in the two years since I made the changes.

I kind of want to move on to other things instead of woof-CE.  I did some work on ROX-Filer, and I want to try to make something very compact that can run in RAM on a Pi zero (command line only).  I did test the raspup build on a Pi zero and it works fine, it's just that a lot of the time people connect to their Pis with ssh and all the graphical X stuff in Puppy would just be a waste of RAM for the kinds of projects that would use a Pi zero.  There are apparently difficulties getting save files unmounted properly, and I wanted to try a different approach keeping the initrd as the root file system.

I don't mind being given credit for the past work I did on woof-CE, but I don't want to be considered a current active member when I want to move on to projects that aren't really Puppy anymore.  I will probably keep watching what is going on like jamesbond3142 does and I can always issue pull requests from my fork of woof-CE if I do have something to share.

**Comment by wdlkmpx** [comment]
It's quite possible to create something very small using the current stuff.
i'm interested myself in creating something that is entirely based on
busybox, so the compat distro pkgs are just a bonus that can get very big..
a working and patched xorg, etc.

A new directory can be created woof-distro/arm/stretch-zero

It's just a matter of deleting many many entries in PKGS_SPECS.. about 90%
95%

The graphical core scripts are all relatively small and easy to identify
and delete, but some of them are also cli scripts. Maybe a new variable in
00build.conf.

But indeed, there is complexity to achieve this simplicity.

The full initrd.gz is currently 2.5Mb.. it eats about 4.5mb in ram.. a
smaller initrd is produced if pupmode = 2 (running ./build.sh -prebuilt in
initrd-progs) 1Mb .gz.. i think. This is also configurable and there is a
config file indeed.

But the initrd is completely destroyed after switching root, so the size is
not a big deal, except that now it's preserved in a new tmpfs.

As the experiment to pivot into the initrd and completely dismantle the
stack wasn't [that] successful, but it does seem to improve the situation
in some cases, right after creating a savefile and rebooting, and notably
in an exfat partition in pupmode 12/13 (if i remember correctly), maybe
that code could be deleted.

**Comment by wdlkmpx** [comment]
I once accidentally installed busybox with all its symlinks, and i started
to notice a strange behavior, and i said to myself what is going on. After
rebooting i saw more errors.

And then i fixed some scripts to improve compatibility with busybox
applets, that's basically what can be done. But there might be no need fix
some scripts because:

busybox in woofce is always cutting edge (rolling release), i already have
1.30, but i'll wait for 1.31 for the next initrd-progs and busybox static
release..

**Comment by wdlkmpx** [comment]
Another thing that can done is move noarch to github and all the extracted
pkgs that contain scripts added to noarch-sources. Move rootfs-packages to
noarch and noarch-sources, move scripts or set of scripts to no arch and
noarch-sources.

That way only the pkgs_specs defines everything, and the pkg manager can
install what's missing making it easier, that will require an active
community i guess, i guess most of the work to unify stuff in
rootfs-skeleton is already done, although keeping everything together makes
it easier to understand and unify everything indeed.

**Comment by wdlkmpx** [comment]
The init script will be more modular, i'll move out big code blocks (for
pupmode 77),code to select pupsave, etc. Easy to fork.

3builddistro will be more modular as well, it was easy for me to adapt
woof-next to call initrd-progs/build.sh -prebuilt and mk_iso.sh, a complete
iso is produced but it stops at switch_root, incomptabilities with current
rootfs-skeleton and the old busybox used in the build.. probably.. have to
read the woof-next git log for some clues.

**Comment by peabee** [comment]
> Proper o acceptable compatibility with ubuntu/debian is a complex issue that will require more work and fixes... and this will make peebee's life a nightmare....

Nightmares I DEFINITELY do not want......

**Comment by peabee** [comment]
Just tried to make a small change to team.md and got:

> You’re editing a file in a project you don’t have write access to. Submitting a change to this file will write it to a new branch in your fork peabee/puppylinux-woof-CE.github.io, so you can send a pull request.

Which suggests I don't have commit rights.....????

**Comment by mavrothal** [comment]
> Which suggests I don't have commit rights.....????

Strange... can you commit in the woof-CE repo?

**Comment by mavrothal** [comment]
Hmmm... looking back, this repo was created by Mick and indeed only 3 people have write access. 
Being the official puppy page I will leave it up to him to decide access by others. 
Hopefully he is reading his mails.
Till then pull requests should do.

**Comment by dimkr** [comment]
> I haven't seen them for a very long time both here and on the forum. I think they have other interest now and they can be safely re-assigned into the "retired" category.

You have my consent. I'm haven't been actively contributing to woof-CE for a very long time, although I'm watching :+1:

**Comment by mavrothal** [comment]
I think the way it is now is pretty straight forward.
Juts shows woof-CE members with commit rights. 
Whoever, does not want to be/show as a member can leave the "organisation" as @woodenshoe-wi did.
Watching is good enough though. Is reassuring to have more eyes looking at changes that are through direct member commits instead of pulls.
